File: assignment/intcharfloat.cpp
#include<iostream>
using namespace std;
class Data_Type
{
    int a;
    char b;
    float c;
    public: void display()
    {
        cout<<"enter the int value \n";
        cin>>a;
        cout<<"enter the name \n";
        cin>>b;
        cout<<"enter the float value \n";
        cin>>c;
        cout<<a<<b<<c;
    }
};
int main()
{
    Data_Type obj;
    obj.display();
}

File: Java Assignment 1/Int_Char_Float.java
import java.util.Scanner;
class Int_Char_Float
{
int num;
char name;
float salary;
void accept()
{
Scanner sc= new Scanner(System.in);
System.out.print("Enter a number: ");
num=sc.nextInt();
System.out.print("Enter a name: ");
name=sc.next().charAt(0);
System.out.print("Enter the salary: ");
salary=sc.nextFloat();
}

public static void main(String[] args)
{
Int_Char_Float obj=new Int_Char_Float();
obj.accept();
}
}


 